File: engine/src/main/java/de/hanno/hpengine/engine/component/InputControllerComponent.java
package de.hanno.hpengine.engine.component;

import java.io.Serializable;

public class InputControllerComponent extends BaseComponent implements Serializable {

    private static final long serialVersionUID = 1L;

    @Override
    public String getIdentifier() {
        return "InputControllerComponent";
    }

    @Override
    public boolean isInitialized() { return true; }
}

File: engine/src/main/java/de/hanno/hpengine/engine/component/InputControllerComponent.kt
package de.hanno.hpengine.engine.component

import de.hanno.hpengine.engine.entity.Entity
import java.io.Serializable

abstract class InputControllerComponent(entity: Entity) : BaseComponent(entity), Serializable {

    companion object {

        private const val serialVersionUID = 1L
    }
}
